src/Microsoft.DotNet.Interactive.Mermaid/UmlClassDiagramExplorer.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Interactive.Mermaid;
+
+public class UmlClassDiagramExplorer
```

Comment:
What makes this an "explorer"?

Reply:
don't know: it was

Reply:
I have renamed that to `UmlClassDiagram`